bundlers/rollup-config/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "@flatfile/rollup-config",
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.mjs",
+  "scripts": {},
```

Do we wanna make this `private` and then release when we're ready or is this g2g?

```suggestion
  "scripts": {},
  "private": true,
```